### ISSUE TITLE
Solution does not build with Visual Studio 2015 Update 3

### ISSUE THREAD

**Issue by orthid** [ms-iot/pid-control-system#6]
After recursive git clone according to README solution does not build.

The solution complains about not finding all references for "Motor":
"Failed to resolve all project references for 'Motor'. The package restore result for '[...]\DemoApp\DemoApp.csproj' may be incomplete."

Trying to build "Motor" standalone results in:
"\pid-control-system\motor\pch.cpp : fatal error C1192: #using failed on 'C:\Users\Daniel\pid-control-system\Debug\Microsoft.Iot.Lightning.Providers\Microsoft.IoT.Lightning.Providers.winmd'"

Installing the Lightning providers manually with NuGet does not help.

**Comment by wind-rider** [comment]
I also encountered this problem.

As far as I remember, I solved it by roughly the following steps: creating a new c++ WinRT component project, copying the relevant source files from the motor project and adding the Lightning NuGet package (I think the package ending in Lightning.Providers is removed from NuGet). Probably this demo app is still referencing an old NuGet package version.

Also look at this tutorial https://developer.microsoft.com/en-us/windows/iot/docs/lightningproviders to see how to use them (there should be something added to the app manifest and the device should run the correct driver model).
